service/gateway/router_test.go

```diff
@@ -146,6 +146,38 @@ func TestRouters(t *testing.T) {
 			shouldMatch:      true,
 			wantedRouterName: getUserBucketsRouterName,
 		},
+		{
+			name:             "Get object metadata router, virtual host style",
+			router:           gwRouter,
+			method:           http.MethodGet,
+			url:              scheme + bucketName + "." + testDomain + "/" + objectName + "?" + model.GetObjectMetaByNameQuery,
+			shouldMatch:      true,
+			wantedRouterName: getObjectMetaByNameRouterName,
+		},
+		{
+			name:             "Get object upload progress router, path style",
+			router:           gwRouter,
+			method:           http.MethodGet,
+			url:              scheme + testDomain + "/" + bucketName + "/" + objectName + "?" + model.GetObjectMetaByNameQuery,
+			shouldMatch:      true,
+			wantedRouterName: getObjectMetaByNameRouterName,
+		},
+		{
+			name:             "Get bucket metadata router, virtual host style",
+			router:           gwRouter,
+			method:           http.MethodGet,
+			url:              scheme + bucketName + "." + testDomain + "?" + model.GetBucketMetaByNameQuery,
+			shouldMatch:      true,
+			wantedRouterName: getBucketMetaByNameRouterName,
+		},
+		{
+			name:             "Get bucket upload progress router, path style",
```

Comment:
Ditto.

Reply:
Pls add list user buckets router ut.

Reply:
The list user buckets router ut is already there, named getUserBucketsRouterName.